spec/rails_helper.rb

```diff
@@ -0,0 +1,33 @@
+ENV["RAILS_ENV"] ||= "test"
+require File.expand_path("../../config/environment", __FILE__)
+abort("The Rails environment is running in production mode!") if Rails.env.production?
+require "spec_helper"
+require "rspec/rails"
+# Add additional requires below this line. Rails is not loaded until this point!
+require "capybara/rspec"
+require "pundit/rspec"
+require 'pundit/matchers'
+Dir[Rails.root.join("spec/support/**/*.rb")].each { |f| require f }
+ActiveRecord::Migration.maintain_test_schema!
+Capybara.register_driver :selenium_chrome do |app|
+ Capybara::Selenium::Driver.new(app, browser: :chrome)
+end
+Capybara.javascript_driver = :selenium_chrome
+RSpec.configure do |config|
+  # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
+  config.fixture_path = "#{::Rails.root}/spec/fixtures"
+  config.use_transactional_fixtures = false
+  config.infer_spec_type_from_file_location!
+  config.filter_rails_from_backtrace!
+  config.include Devise::Test::ControllerHelpers,  type: :controller
+  config.include Devise::Test::ControllerHelpers,  type: :view
+  config.include Devise::Test::IntegrationHelpers, type: :feature
+  config.include Rails.application.routes.url_helpers
```

Comment:
that's pretty big module. Define implicitly the type of spec files where it should be included